data/src/generate_phones_summary.py

```diff
@@ -53,7 +53,8 @@ def main() -> None:
         if file_path.endswith(".md") or file_path.endswith("tsv"):
             continue
         with open(f"{path}/{file_path}", "r", encoding="utf-8") as phone_list:
-            num_of_entries = sum(1 for line in phone_list)
+            # We subtract one for the final newline.
+            num_of_entries = sum(1 for line in phone_list) - 1
```

Comment:
I just noticed this, but this script counts comments as "entries" (which isn't what we want, I should think). I guess the line should be `num_of_entries = sum(1 for line in phone_list if line[0] != "#") - 1` or something to that effect.

Reply:
+1. I'll create an issue.